app/tests/test_routes.py

```diff
@@ -31,6 +31,15 @@ def test_home(self):
 
         self.assertEqual(response.status, '308 PERMANENT REDIRECT')
 
+    @patch('app.search.gleaner.GleanerSearch.get_total_count')
+    def test_total_count(self,gleaner):
+        gleaner.total_count = self.mock_result_set
```

Comment:
Instead of setting this to `self.mock_result_set`, you can set it to a number of your choice, and you can also test that you get that number back, in the response body.

Reply:
That makes sense!
Thanks!